src/lib/hooks.js

```diff
@@ -125,6 +125,10 @@ const api = {
 		updateIframeGuestData({ phone });
 	},
 
+	unsetGuestPhone() {
+		updateIframeGuestData({ phone: 'delete' });
```

Comment:
@AlexanderKanakis this method sets the phoneNumber to be the text string 'delete'.  Is it possible to just unset the number, so that there is no entry in the db for it?

Reply:
We use that 'delete' string to unset the number on the server and db. Check out the Rockey.chat PR https://github.com/WideChat/Rocket.Chat/pull/1142

Reply:
@AlexanderKanakis I don't love passing a string around to delete a value.....  it just seems not best practice.  Is there anyway to set this value to 'None' or even an empty string instead?  Would that work, or do you have to pass a string?

Reply:
Passing an empty string creates an issue similar to what @Shailesh351 noted; we unset all other instances of the same customfield in the DB. We need a way to make sure that the change only happens to that specific user.

Reply:
If we are not happy with it, I will explore alternatives like `null`.

Reply:
Based on @Shailesh351 comment on the accompanying RC PR, seems like 'delete' might be the way to go?